doc/nrf/links.txt

```diff
@@ -506,7 +506,7 @@
 
 .. _`TinyCBOR`: https://intel.github.io/tinycbor/current/
 
-.. _`BH1749NUC-E`: https://www.rohm.com/datasheet/BH1749NUC/bh1749nuc-e
+.. _`BH1749NUC-E`: https://d1d2qsbl8m0m72.cloudfront.net/en/products/databook/datasheet/ic/sensor/light/bh1749nuc-e.pdf
```

Comment:
This one looks suspicious. I see that the original link is giving a 301 to this location, but a random `<hash>.cloudfront.net` URL looks more likely to break than this alternative:

https://fscdn.rohm.com/en/products/databook/datasheet/ic/sensor/light/bh1749nuc-e.pdf